src/main/java/com/mongodb/jdbc/MySQLDatabaseMetaData.java

```diff
@@ -10,6 +8,7 @@
 import java.sql.Types;
 import java.util.ArrayList;
 import java.util.regex.Pattern;
+import org.bson.*;
```

Comment:
I may have introduced this wildcard import last PR. I thought I got rid of all of them, but must have missed this one. Do you want to expand it to just the imports we need? Your call.

Reply:
Oh, I didn't even notice this. I'll fix it! I think it did come from the last PR.